### PR TITLE
Include pfSense custom Dictionary

### PR DESCRIPTION
**Pull request by Augustin-FL**

### CODE REVIEW

File: share/dictionary/radius/dictionary.pfsense [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # -*- text -*-
-##############################################################################
+# Copyright (C) 2015 The FreeRADIUS Server project and contributors
```
Comment (alandekok) [inline share/dictionary/radius/dictionary.pfsense]: That should really be 2019...

Comment (arr2036) [reply share/dictionary/radius/dictionary.pfsense]: and you probably want to $INCLUDE the dictionary in the top level dictionary file

**Comment by Augustin-FL** [comment]
Hello,

It's done ! Sorry for these beginner mistakes.

**Comment by arr2036** [comment]
Can you rebase your changes into a single commit, and force push to your PR branch.

**Comment by Augustin-FL** [comment]
The travis build is failing because of an expired certificate...it seems like it isn't failing because of my PR (or am I mistaken ?)

**Pull request merged**